src/main/resources/assets/ironchest/lang/fi_FI.lang

```diff
@@ -0,0 +1,177 @@
+################
+#    Chests    #
+################
+
+tile.ironchest.chest.iron.name=Rautainen arkku
+tile.ironchest.chest.gold.name=Kultainen arkku
+tile.ironchest.chest.diamond.name=Timanttinen arkku
+tile.ironchest.chest.copper.name=Kupari arkku
+tile.ironchest.chest.silver.name=Hopea arkku
+tile.ironchest.chest.crystal.name=Kristallinen arkku
+tile.ironchest.chest.obsidian.name=Obsidiaani arkku
+tile.ironchest.chest.dirtchest9000.name=Multa arkku 9000!
+
+item.ironchest.chest.iron_gold.name=Parannus: raudasta kultaisen
```

Comment:
raudasta kultaiseen or raudasta kultainen?

raudasta kultaisen is kind of funny. Same comment for all.

Reply:
raudasta kultaisen

Reply:
Thats bettered?